CareKit/CareKitTests/Contact/TestSimpleContactViewSynchronizer.swift

```diff
@@ -85,6 +85,6 @@ class TestSimpleContactViewSynchronizer: XCTestCase {
         viewSynchronizer.updateView(view, context: .init(viewModel: nil, oldViewModel: nil, animated: false))
         XCTAssertNil(view.headerView.titleLabel.text)
         XCTAssertNil(view.headerView.detailLabel.text)
-        XCTAssertEqual(view.headerView.iconImageView?.image, UIImage(systemName: "person.crop.circle"))
+        XCTAssertEqual(view.headerView.iconImageView?.image, UIImage(systemName: "questionmark.circle.fill"))
```

Comment:
I think we may want to keep the default image as "person.crop.circle"

Reply:
Yes, and we are using that. I introduced the questionmark icon there so that we can use it as a universal placeholder.

If you look at OCKHeaderView+Updatable, you'll see the updated logic. We are using OCKHeaderView for various purposes - Events, Tasks, Contacts, and TaskCategories. If the clear out and not configured state was using a person placeholder, it would look strange in the TaskCategory list.

The Contact list will not be impacted by this change. The difference is in the configuration. If you configure the OCKHeaderView with a OCKAnyContact that doesn't have any asset - the image will be rendered as a person placeholder. In case of the OCKAnyTaskCategory, a question mark image will be used.

Let me know if this is desirable logic or what other solution would you suggest.